erizo/src/erizo/rtp/RtpPaddingGeneratorHandler.cpp

```diff
@@ -100,6 +100,7 @@ void RtpPaddingGeneratorHandler::sendPaddingPacket(std::shared_ptr<DataPacket> p
   RtpHeader *rtp_header = reinterpret_cast<RtpHeader*>(padding_packet->data);
 
   rtp_header->setSeqNumber(sequence_number.output);
+  rtp_header->setMarker(false);
```

Comment:
could we set the marker bit inside RtpUtils::makePaddingPacket please?

Reply:
I'm sorry idk why i messed up here